### ISSUE TITLE
tools/process - error: Multiple Plugin Paths Provided

### ISSUE THREAD

**Issue by winterheartlee** [AuburnSounds/Dplug#318]
Hi guys, I've built the plugin host from tools/process successfully but when I try to use process.exe with the arguments specified, I keep getting the error:

error: Multiple Plugin Paths Provided

I initially popped the .dll into the root folder and just used plugin.dll for the argument but I've also tried giving the plugin argument as an absolute path and still get the same error...

Any ideas whats wrong? I hope its ok to ask these sort of questions here as I just have nowhere else to look :S 

Thanks

**Comment by winterheartlee** [comment]
Hi! Thanks for the quick response!
For now I'm just running: 

`process [-i input.wav] [-o output.wav] plugin.dll`

And I have popped in a wav file in the root folder named input.wav (just for the sake of clarity) and also popped in a .dll named plugin.dll (just renamed your distort.dll example that I built, but I've also tried using other working plugins to eliminate an issue with the build)

I've tried running a full command of all the arguments as well but still get the same error.

**Comment by p0nce** [comment]
`[ arguments ]` means the argument is optional.
The command-line you should use instead is:
`process -i input.wav -o output.wav plugin.dll` without the brackets.

With these brackets, `process` believes "[-i" is the path to a plugin, and errors because then you have provided several plugin paths.

**Comment by winterheartlee** [comment]
Ah yes, sorry I did try that initially, but I was getting the error:

error: input.wav: The system cannot find the file specified.

So I thought I was just misunderstanding the arguments when running like that...

Just played around again and removed the .wav extension from input.wav and it seems to process now!
Albeit the processed file was completely silent and somehow double in size (double sample rate maybe?) lol, but I'll work with it now to try and get it running properly for my needs...

Thanks for much for your help! So sorry to have bothered you for such a simple solution, but after hours of trying to build it (understanding the VST2 SDK issues), I just got stumped with this...

Cheers!
